cfme/rest/gen_data.py

```diff
@@ -403,3 +417,28 @@ def _finished():
             logger.warning("Failed to delete arbitration settings.")
 
     return response
+
+
+def orchestration_templates(request, rest_api, num=2):
+    collection = rest_api.collections.orchestration_templates
+    body = []
+    for _ in range(num):
```

Comment:
@mshriver The purpose is to create unique name and description for each test and just number would not help, so I don't see any advantage in including number in addition to unique string. Using underscore where value is not needed is idiomatic in python.